### PR TITLE
Use maven instead of putting jars in lib

### PR DESCRIPTION
**Pull request by amalakar**
Made the following changes:
1. Use maven instead of bundling jar, makes it easier to publish to maven repo, or load in intellij or eclipse
2. Format code
3. Make the README a markdown

### CODE REVIEW

File: src/main/java/com/ooyala/api/OoyalaAPI.java [hunk 1]
```diff
@@ -295,61 +300,64 @@ public String handleResponse(HttpResponse response) throws ClientProtocolExcepti
...
+     * parameters nor a body (requestBody)
      *
      * @param HTTPMethod The HTTP method
      * @param requestPath the request path
      * @return The response from the server as an object of class Object. Must be casted to either a LinkedList<String> or an HashMap<String, Object>
-     * @throws IOException 
-     * @throws ClientProtocolException 
-     * @throws NoSuchAlgorithmException 
-     * @throws HttpStatusCodeException 
+     * @throws IOException
```
Comment (Pepegasca) [inline src/main/java/com/ooyala/api/OoyalaAPI.java]: According to the change on the function signature, this function throws IOException but the function documentation keeps the other three type of exceptions.

Comment (amalakar) [inline src/main/java/com/ooyala/api/OoyalaAPI.java]: I haven't changed the exceptions thrown. They are just in the new line. Trying to limit number of characters per line to 120.

Comment (Pepegasca) [inline src/main/java/com/ooyala/api/OoyalaAPI.java]: The function documentation mentioned about ClientProtocolException, but the new function signature not contains this exception.

**Comment by amalakar** [comment]
@Pepegasca removed unnecessary exception from function signature. Please take a look.

**Pull request merged**